src/app/auto-correct/auto-correct.component.css

```diff
@@ -29,7 +29,7 @@
 
 @media screen and (max-width: 1133px) {
   p.mean {
-    margin-left: 3.5%;
+    margin-left: 2%;
```

Comment:
margin is something related to resultcomponent.
**Take this case:**
in future, we might use the same auto correct component in some other component, then we might require different margins there.
**what should you do**
These styles should be binded to `<app-autocorrect>` element in results component.because these styles define the position of auto correct component **"in"** resultcomponent, it is an external behavior to the auto-correct component.